prac_06/languages.py

```diff
@@ -0,0 +1,19 @@
+"""Prac06 languages.py
+Estimated time: 20min
+Actual time: 13min
+"""
+
+
+from prac_06.programming_language import ProgrammingLanguage
+
+
+python = ProgrammingLanguage("Python", "Dynamic", True, 1991)
+ruby = ProgrammingLanguage("Ruby", "Dynamic", True, 1995)
+visual_basic = ProgrammingLanguage("Visual Basic", "Static", False, 1991)
+print(python)
+
+languages = [python, ruby, visual_basic]
+
+for language in languages:
```

Comment:
Have you forgotten the `"The dynamically typed languages are: "` string here?

Reply:
Good catch... Fixed.